client/src/panels/welcome_mat/welcome_mat.yaml

```diff
@@ -8,9 +8,9 @@ no_wrap_year:
   handlebars_partial : true
   text: <span class='text-nowrap'>({{year}})</span>
 years_ago:
-  transform: [handlebars]
+  transform: [handlebars, markdown]
   en: |
-    {{actual_hist_years_apart}} years ago ({{oldest_hist_spend_year}})
```

Comment:
As an alternative to defining a new handlebars helper, you can make use of the handlebars `{{#if}}{{/if}}` block here. Something like:
```
  {{actual_hist_years_apart}} year{{#if plural_years}}s{{/if}} ago ({{oldest_hist_spend_year}})
```

Where `plural_years`, or whatever you name it, is a boolean you calculate in JS and pass in as a `text_maker` arg.